index.html

```diff
@@ -105,12 +96,13 @@ <h3>DDD - CQRS - CQRS/ES</h3>
                 <small>with the permission of the author</small>
             </section>            
             <section>
-                <h3>        Domain & Ubiquitous Language</h3>
-                -
-                <h3>Two brothers always with us</h3>
-            </section>            
-            <section>
-                <h3>pic-matrioska-todo - Overall image from business to Ide and Vice Versa</h3>
+                <h3>Overall image from business to Ide and Vice Versa</h3>
+                <img
+                class="r-stretch"
+                src="img/all-concepts.jpg"
+                width="500"
```

Comment:
Why the width? It could cause issues with the layout. It should be fixed like this:
```suggestion
                class="stretch"
                src="img/all-concepts.jpg"
```